### DISCUSSION TITLE
Optimale Einstellungen im Power Limiter für den Akku

### DISCUSSION THREAD

**Discussion by frankydee1980** [hoylabs/OpenDTU-OnBattery#263] [Q&A] [answered]
Hallo Zusammen,

bei mir läuft openDTU onbattery seit einem Monat ohne größere Probleme mit Victron Laderegler und 2x 12V AGM Batterien im 24V Verbund.
Meine Frage ist, was sind eigentlich die optimalen Einstellungen für DC Start und Stop? Laut Internet sollten die Batterien nicht unter 11,8V bzw. 23,6V entladen werden. Habe die Stop Spannung also bei 24,4V gesetzt. Die Start Spannung auf 27,6V, da das die Erhaltungspannnung ist. Richtig so?

Viele Grüße
frankydee

**Comment by uli-bs** [comment]
Bei mir ist es ein 24V LiFePO4 Akku, die Spannungswerte habe ich auf 24.0V und 24,5V eingestellt, sonst geht der Akkubetrieb ja erst bei voller Batterie los. 
Damit der Akku nicht weiter entladen werden kann (er speist neben dem WR noch weitere Elektronik) hängt noch eine Ladeschaltung mit Konstantspannung / -strom (23,5V / max. 300mA) dran, das deckt erforderlichenfalls den "Ruhebedarf".

**Comment by neolauren-cmd** [comment]
Brauche ich von Victron eigentlich den Smart Battery Protect oder würde das die OpenDTU Firmware regeln und ich kann mir das Teil sparen?

Gruß Neo

**Reply by kloppy1984** [reply, marked as answer]
Ich würde dir ein Battery Protect empfehlen!

(Laut Handbuch sollte an den Battery Protect kein Wechselrichter direkt betrieben werden.
Was bedeutet, es muss über ein Relais deinen Wechselrichter schalten!)

Damit hast du Theoretisch 3 Maßnahmen die deine Batterie von Entladung schützen:
1. Open DTU schaltet den Wechselrichter ab, bei zu wenig Spannung.
2. Schaltet dein Wechselrichter nicht ab (weil die Signale der DTU nicht ankommen) schaltet den BatteryProtect ab
3. sollte dieses nicht abschalten, hast du noch das BMS, was deine Batterie schützt.

Ich hab meinen DTU-Stop bei 23,8.V eingestellt, Start bei 25,8V
BatteryProtect schaltet bei 23,6V ab, Start bei 25,6V
BMS schaltet bei 23,2V ab.

**Comment by neolauren-cmd** [comment]
Hallo Zusammen,
ich habe nochmal eine Frage zum Battery Protect. Ich dar ja den Plus nicht direkt schalten der zum WR geht. Würde denn hinter dem BP ein Relais 24V/120A gehen und damit schalte ich bei beiden Eingängen meines HM-800 den Plus?
Hab jetzt soviel gelesen, die ihren BP geschossen haben, weil sie ihren WR direkt damit verbunden haben und hab jetzt voll die Panik, was zu schießen und vielleicht einen Brand zu riskieren.

Danke für eure Tipps.

Gruß 
Neo

**Reply by kloppy1984** [reply]
Moin, also ich habe den Battery Protect direkt angeschlossen. Bisher ohne Probleme.

Mit einem Relais bist du auf der sicheren Seite.

Den Battery Protect habe ich am Anfang ein, zweimal auf Funktion getestet.
Mittlerweile lasse ich den Wechselrichter per Software ausschalten, wenn Battery mal leer sein sollte. Somit kommt der Battery Protect als zweite Instanz zu Einsatz. Er wird aktuell nicht beansprucht, da er nicht abschalten muss.

**Reply by swingstate** [reply]
Ich schalten meinen WR auf der AC Seite mittels Smarthome (home assistant) + Shelly Plug. Die Batteriespannung lesen ich mit einem Victron Smartshunt + ESPhome aus. 
Damit bleibt die DC Seite immer verbunden. So läuft das Ganze schon seit 6 Monaten ohne Probleme.

**Reply by neolauren-cmd** [reply]
Was heißt per Software ausschalten? Wie hast du das realisiert?
Also wenn ich den Plus der zum WR geht mit einem Relais schalte, dann wäre die Last auf dem Relais, nicht auf dem BP oder? Hab  halt soviel jetzt gelesen, dass bei vielen der BP durchgebrannt ist und das will ich dann mit dem Relais vermeiden, wenn es denn so klappt. 

Gruß 
Neo

**Reply by kloppy1984** [reply]
Per Software heißt, openDtU-Battery schaltet den Wechselrichter aus.
Er bleibt aber DC und AC Seitig am Netz angeschlossen.

**Reply by swingstate** [reply]
Der Victron Shunt hat via Ve.Direct einen ESP, auf dem ESPhome geflasht ist, angeschlossen.

Dieser ESPhome ist in Homeassistant Integriert, womit ich dann den Ladezustand, Spannungen etc. live auslesen kann.

Auf Basis der Spannung die der ESPhome live reported schaltet eine Automation in Homeassistant die Steckdose in welcher der Wechselrichter steckt.

Bei 25,8V wird die Steckdose ausgeschaltet. Die Batterie bleibt weiterhin verbunden.
Bei 28,35V wird eingeschaltet.

**Reply by swingstate** [reply]
Man kann meinen und den Ansatz von @kloppy1984 kombinieren, um ein Fallback zu haben.

**Comment by neolauren-cmd** [comment]
Hab jetzt glaub ich schon den zweiten Hoymiles geschossen. Ich gehe von der Batterie direkt über den BP zum Hoymiles. Der BP kommt sofort Kurzschluss und der WR macht nix mehr. Was mache ich falsch? Beim ersten Mal anscheinend Plus und Minus verwechselt, aber jetzt schon wieder? Ich brauche Hiiiilllffe....
Liegt das am Eingngsstrom von der Batterie? Ich habe eine 24V Batterie 100AH. Auf dem HM-800 steht max Conntinous Input Current 2x 12.5A d.c. Liegt das da dran? Bei euch funktioniert es doch auch oder?

Update: Der zweite WR funktioniert noch. Ich traue mich nicht mehr den an die Batterie zu schließen..... Kann das am BP liegen? Soll ich den mal weglassen?

**Reply by kloppy1984** [reply]
Bei mir geht der Plus über den BP zum HM-600.
Masse direkt angeschlossen.
Dazwischen halt noch eine Sicherung.

Wenn du plus und Minus verwechselst, kannst du dabei schnell den Hoymiles killen!

Vielleicht helfen dir diese Bilder weiter.

![Screenshot_20230401_211935_Chrome](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/63321205/57b3e990-a3f9-4485-8aac-0631d7a549df)
![Screenshot_20230330_085931_Amazon Shopping](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/63321205/513f2dc2-6a39-4187-a938-26449251d150)

**Reply by kloppy1984** [reply]
Je nachdem ob die einzelne direkte Kabel oder Y Kabel zum WR hast, kann es gut sein, das Schwarz dein Plus ist!

**Reply by schlimmchen** [reply]
Aus einem Solarmodul kommend ist dieser Stecker "Plus":
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/3578416/08a05cc4-9f15-4e3c-bcac-fdfb6bd7c51c)

Den Rest kannst du dir in Ruhe ableiten und aufschreiben.

Den Wechselrichter direkt an eine Batterie anzuschließen ist nicht ratsam. Im Wechselrichter sind Kondensatoren mit hoher Gesamtkapazität, und da fließt beim Zuschalten der Batterie ein extrem hoher Strom für kurze Zeit. Es ist ratsam, die Kondensatoren zuerst über einen Widerstand (1-5 Ohm) Aufzuladen und dann die Batterie zuzuschalten. Dazu gibt es an vielen Stellen im Internet Diskussionen/Anleitungen dazu.

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/3578416/5a3dc9f7-d21c-405e-9afa-97d000d9bb80)


Was der hohe Strom mit dem Battery Protect macht, kann ich nicht sagen. Der scheint das ja zu bemerken und versucht einzugreifen. Mit Vorladewiderstand wird das nicht passieren.

**Comment by neolauren-cmd** [comment]
Also Plus und Minus passt jetzt. Hab aber trotzdem nicht verstanden, warm es bei manche geht, die über den BP den Plus schalten und dann an den WR und bei machen nicht. Würde es was bringen, wenn ichvan einen Eingang vom WR ein PV Modul anschließe und beim anderen den Akku, zwecks den Kondensatoren?

**Reply by spcqike** [reply]
wahrscheinlich nicht, da es zwei unabhängige Eingänge sind. 

Außerdem kommt dann die ganze Regelung durcheinander, da die openDTU keinen Mischbetrieb vorsieht.

wie anderswo bereits geschrieben: Lade deine Eingänge vorsichtig auf und Trenne den HM nicht permanent von der Batterie. Schalte den HM per Software aus.

der BP ist meiner Meinung nach nur die letzte Reißleine die greifen sollte, bevor die Batterie tiefenentladen und damit beschädigt wird. weder das BMS eines Lithium Akkus, noch der BatteryProtect sollten die Batterie überhaupt trennen. (außer als aller letzte Absicherung eben).

Die OpenDTU-onBattery sollte deinen Wechselrichter per Software ausschalten.

**Reply by neolauren-cmd** [reply]
Okay. Wo muss ich diese Widerstände anschließen? Ich gehe doch mit 6mm Kabelquerschnitte zum WR.

Und warum funktioniert es bei manchen und hier nicht? Sollte ich eventuell zuerst die AC Seite einstecken und dann erst die DC Seite?

**Reply by neolauren-cmd** [reply]
Bzw. ich habe jetzt beide Eingänge vom WR mit Solarmodulen verbunden? Soll ich das mal noch eine Stunde lassen und dann den Akku anschließen? Wären die Kondensatoren dann aufgeladen?

**Reply by spcqike** [reply]
Die Widerstände sollten nicht dauerhaft zwischen Batterie und wechselrichter hängen. Immerhin fällt über ihnen eine eine gewisse Spannung ab die gleichzeitig Verluste bedeutet. 

Bei 1 Ohm und 10A immerhin 10V. Und damit 100W. 
bei 2 Ohm entsprechend 20V. Damit bliebe dem wechselrichter nicht mal mehr genug spannung zum laufen. 

Es gibt Schaltungen die den Widerstand nur am Anfang nutzen und kurz nach zuschalten der Batterie Brücken, damit er nicht dauerhaft im Stromkreis hängt. 

Warum es bei anderen klappt kann icu nicht sagen. Glück? Chargen Schwankungen? Weniger ein/aus Zyklen? 

Es ist ja auch nicht gesagt, dass der wechselrichter direkt beim ersten anschließen der Batterie knallt. Die Kondensatoren können schleichend beschädigt werden. 

Ob an AC zuerst was hängt sollte egal sein. Der Kondensator hängt direkt am DC Eingang und lädt sich eben so schnell auf, wie geht. Ein Paneel liefert anfangs kaum Strom. Eine entsprechende Batterie immer >100A

> Wären die Kondensatoren dann aufgeladen?

das dauert keine Sekunden. 

Kondensator aufladen ist das, was es an der Schuko Steckdose funken lasst, wenn billige Netzteile eingesteckt werden ;)

**Reply by neolauren-cmd** [reply]
Heißt das jetzt, nachdem die Platten dran waren, kann ich jetzt schnell den Akku anschließen und es sollte funktionieren?

**Reply by kloppy1984** [reply]
Jip, aber ohne 230V, sonst zieht der Wechselrichter dir wie Kondensatoren wieder leer  😅

**Reply by spcqike** [reply]
> aber ohne 230V, sonst zieht der Wechselrichter dir wie Kondensatoren wieder leer 😅

entweder das, oder den Ausgang per Software aus machen.

**Reply by neolauren-cmd** [reply]
Dann probiere ich das, mal schsuen ob mein BP wieder Kurzschluss zeigt.

**Reply by neolauren-cmd** [reply]
Hat geklappt, jetzt läuft es. :-) wenn ich AC trenne, kann ja nichts passieren oder? Die Kondensatoren bleiben ja weiter von der DC Seite inter Spannung oder sehe ich das falsch?
Ihr müsst mir jetzt nur noch sagen, was och genau benötige, um nach einen DC Ausfall nicht wieder Kurzschluss zu bekommen. Hab nun was mit Widerstand oder Step Down oder Step up oder DC DC Wandler gelesen. Was ist denn nun das Beste?

**Reply by spcqike** [reply]
Ein DC DC wandler ist das falsche. (Außer er limitiert auch den Strom. Aber eigentlich brauch man den nur bei 12V Batterie Systemen) 

guck mal hier 
https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/145#discussioncomment-5595340

**Comment by neolauren-cmd** [comment]
Kurze Rückmeldung, bei mir läuft jetzt soweit alles.

Was mir aber aufgefallen ist, ich habe am HM-800 fest 250W eingestellt, aber der WR schwankt total. Auf der DC Seite liegen teilweise 280W an und auf der AC Seite nur 120W. 100 Watt Verlust? Ist das normal oder hab ich irgendwas durch meine erste Misere geschossen? Batteriespannung ist konstant 26.3 V. Ach liegen beide WR Eingänge teilweise 40V auseinander, obwohl die kurzen Kabellängen bei beiden gleich ist.. 

Gruß 
Neo

**Reply by spcqike** [reply]
Was sagt denn der Wirkungsgrad in der Zeit? Ich meine andere Leute hätten auch schon berichtet, dass der WR gelegentlich auf 50% einbricht und nicht alleine wieder auf >90% kommt. Da half wohl nur ein Neustart des WR. (Den man im Batterie betrieb wohl sowieso regelmäßig machen sollte. Kann man im Web UI auch einstellen)

**Comment by kloppy1984** [comment]
![Screenshot_20230705_222317_Chrome](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/63321205/b56af5c5-5fcb-4c30-8fa6-121bcc6714d1)
![Screenshot_20230705_222342_Chrome](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/63321205/302d9c39-c850-4414-8a69-0e1fc08032da)


Bei mir sieht das so aus.
95% Wirkungsgrad gesamt.

Schwanken tuen Sie eigentlich immer...

**Comment by neolauren-cmd** [comment]
Hab jetzt mal neugestartet. Komischerweise ist die Anzeige noch immer seit 10 Minuten gelb und er bringt keine Leistung, obwohl 26,2 V anliegen. Die grüne LED blinkt aber. Nochmal neustarten?

**Reply by neolauren-cmd** [reply]
Jetzt ist er von alleine wieder gekommen.  Wirkungsgrad schwankt zwischen 40 und 95%. Das ist doch nicht normal oder? Fest sind 250W eingestellt.

**Reply by kloppy1984** [reply]
Wie gut oder schlecht ist deine 230V Leitung / Steckdose am Wechselrichter?

Ich befürchte das du beim verloren dir einen Eingang zerschossen hast

**Reply by neolauren-cmd** [reply]
Komischweise hsbe ich bei Einstellung von 100 W am WR keine Schwankungen. Es geht erst bei 200W los.
Oh nein, nicht noch ein WR. Also sind die Schwankungen vom Wikungsgrad 40% auf 95% und wieder zurück, dann nicht normal oder?

**Reply by neolauren-cmd** [reply]
Die 230V Leitung ist die originale vom WR. Da sollte eigentlich nichts sein.

**Reply by Manos1966** [reply]
> Jetzt ist er von alleine wieder gekommen. Wirkungsgrad schwankt zwischen 40 und 95%. Das ist doch nicht normal oder? Fest sind 250W eingestellt.

@neolauren-cmd Mein HM-1500 hat es manachmal auch gemacht, Wirkungsgrad 50-60%. 
Nur ein Neustart korrigiert die Situation. 
Ich habe festgestellt, es passiert wenn:
 1- die Spanung der Batterie ist zu niedrig ist aufgrund des hohen Verbrauchs (drop down voltage)
 2- der Wechselrichter stoppt
 3- Spannung geht wieder hoeher
 4- Wechselrichter startet mit der Produktion wieder  
 5- die Spanung der Batterie ist zu niedrig ist aufgrund des hohen Verbrauchs (drop down voltage)
 6- gehe wieder zu Punkt 2...

Seitdem ich die Batterie Spannung besser kontrolieren kann (sowie die Drop-down Berechnung) habe ich dieses Problem nicht mehr gesehen.


Zum Thema zweiter Wecheslrichter: Was spricht dagegen, eine zweite OpenDTUonBattery fuer die Steuerung des zweiten Wechelrichters zu installieren
Wenn die Update (Sekunden) Einstellungen unterschiedlich sind, dann sollten die zwei OpenDTUs, ueber die Zweit, das Gleichgewicht erreichen.

**Reply by kloppy1984** [reply]
Wahrscheinlich ein 24V System?
Bei einen 48V System schwankt die Spannung nicht so stark!

**Reply by Manos1966** [reply]
Ich habe es sowohl bei einer 25 V Batterie als auch bei einer 2x25 = 50V Batterie erlebt (das BMS einer Batterie schaltet ab als erstes, dadurch wird ein 25V Batterie)

**Reply by kloppy1984** [reply]
Erkläre mir bitte wie das funktionieren soll.
Sind beide in Reihe geschaltet, hast du -> 24V +24V. Steigt bei einer Batterie das BMS aus, geht dort nichts mehr durch. Also hast du 0V. Weil die nichts durchlässt. 

So wie du es schreibst, wird aus einer 48V Batterie eine 24V... Das funktioniert so nicht ;-)

**Reply by Manos1966** [reply]
@kloppy1984 du hast Recht, ich habe mich total geirrt. 
Der Fall den ich beschriebn habe, konnte nur bei zwei parallel geschalteten Batterien passiert sein... und dann waeren es weiterhin 25V.
Dann habe ich keine Erklaerung, warum ich diese "dramatische" Spanungssenkungen registriert habe:
![Big Drop Voltage](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/d46d0289-4dd4-4297-ba3a-a0ebe7eeb2cb)

Die Probleme mit dem Wirkungsgrad habe ich nur nach einer deutliche Senkung der Spannung (kurz vor 12:00, oberste Grafik zeigt die Spanung und unterste den Wirkugsgrad)
![Wirkung 52% C](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/9ca9bc2e-dd7a-41c3-86cb-a756f7a7ecdc)

**Comment by neolauren-cmd** [comment]
Wahrscheinlich werd ich den WR nochmal umtauschen. Komischweise ist dieses Phänomen nur, wenn ich 250W einstellen. Höher und tiefer hab ich den normalen Wirkungsgrad von ca. 95%.

Noch eine andere Frage, da ich ja zwei WR nutze und der Power Limiter bisher nur einen unterstützt, gibt es einen Befehl für Node-red, damit ich den WR neustarten kann? Da ich ja meinen großen über die Nulleinspeisung steuere, kann ich das nicht direkt über die OpenDTU machen. 
Daher meine Frage, den anderen automatisiert über einen Befehl neuzustarten.

Gruß 
Neo

**Reply by neolauren-cmd** [reply]
Hallo, 

ich wollte nochmal nachfragen, ob es möglich ist per Node-red den WR neuzustarten bzw. gibt es Neuigkeiten bei dem Projekt; einen zweiten WR über den Power Limiter zu regeln? Da wäre ja die Option mit drinnen und ich könnte auch meinen 2ten WR mit Akku oer Nulleinspeisung regeln. 

Gruß 
Neo

**Reply by helgeerbe** [reply]
Hi @neolauren-cmd mir fehlt im Moment einfach die Zeit mich wirklich mit dem Thema zweiter Wechselrichter, bzw. zweiter Victron Charger zu beschäftigen. Wenn das jemand angehen möchte, da unterstütze ich gerne.

Neustart über node red sollte gehen. Hier mal ein passender curl Aufruf: `curl -u "User:Passwort" http://10.1.1.98/api/power/config -d 'data={"serial":"11418308xxxx","restart":true}'`

Nutze einfach einen http-request node. Beispiel: https://stevesnoderedguide.com/node-red-http-request-node-beginners

**Reply by neolauren-cmd** [reply]
muss ich den kompletten curl Aufruf in die url eingeben? Ich bekomme da eine Fehlermeldung oder wie muss ich den http request ausfüllen?

**Reply by helgeerbe** [reply]
Ich würde als erstes einen function node mit dem folgenden Inhalt erzeugen:

````
msg.payload = {'serial':'11418308xxxx', 'restart':'true'};
msg.headers = {};
msg.headers['content-type'] = 'application/json';
return msg;
````

Der Ausgang vom Function Node geht in den http request node:
- Methode auf `POST`
- Url auf `http://10.1.1.98/api/power/config`
- Basic Auth anklicken und 
  - username: admin
  - password: supergeheim


Ist jetzt nicht getestet. Aber google sollte dir da auch weiterhelfen können.

**Reply by neolauren-cmd** [reply]
Klappt leider nicht. In der Ausgabe kommt: msg.payload : string[59]
"{"type":"warning","message":"No values found!","code":1002}"

Ich schau mal, ob ich was bei google finde, trotzdem danke.

**Reply by neolauren-cmd** [reply]
@helgeerbe ich komme leider nicht weiter. Vielleicht kannst du hier nochmal helfen.

Wenn ich das in den Browser eingebe, müsste doch auch schon was passieren oder nicht?:

http://meineIP/api/power/config -d 'data={"serial":"meineSERIAL","restart":true}'. 

Ich bekomme nur die leer OpenDTU Seite zu sehen, ohne Inhalt. Daher wird es denke ich auch nicht mit Node-Red funktionieren. Könntest du es nochmal prüfen?

Danke Dir.

**Reply by neolauren-cmd** [reply]
Hallo @helgeerbe, konntest du den curl nochmal testen? Ich kann machen was ich will, bekomme es leider nicht hin...

Gruß 
Neo

**Reply by Phantomias2006** [reply]
Hi @neolauren-cmd ,

ich hatte mich am WE mal mit der API gespielt da ich den ESP selber neu starten wollte und es mittels MQTT aktuell nicht möglich ist...
Da du Node-Red verwendest würde es sich anbieten, wenn du den WR mittels MQTT neu startest. Gibt es einen Grund warum du es über die Web-API machen möchtest?

Der Befehl für die WebAPI ist ein POST befehl. Das mit der BrowserUrl im Browser geht nur mit GET befehlen. Also dass das im Browser nicht funktioniert ist normal.

Da ich es mittels NodeRed auch nicht geschafft hatte einen Befehl zu senden und dieselbe Fehlermeldung wie du erhalten hatte hab ich mir erst mal in Postman einen funktionierenden Request zusammen gebaut. Dazu musste ich mich im Source Code einarbeiten.
Folgendes habe ich aus dem Gedächtnis rekonstruiert. Ich kann dir aber am Abend im Detail nachsehen:

`curl --location 'http://10.1.1.98/api/power/config' \
--header 'Content-Type: text/plain' \
--header 'Authorization: Basic dXNlcjpwYXNzd29ydA==' \
--data 'data={'\''serial'\'':'\''11418308xxxx'\'', '\''restart'\'':'\''true'\''}'`

Der Content-Type muss zwingend auf "text/plain" stehen und nicht auf "application/json"
Im payload muss "data=" am Anfang stehen. 

Meine Empfehlung wäre, dass du dir einen funktionierenden Request mittels Postman zusammenbaust. Dann kannst du den Curl dafür ansehen und versuchen es in NodeRed nachzubauen. 

Ich habe es in NodeRed selbst noch nicht versucht seitdem ich mich mit Postman gespielt hatte.
Du kannst aber folgendes Testen:
`msg.payload = "data={'serial':'11418308xxxx', 'restart':'true'}";
msg.headers = {};
msg.headers['content-type'] = 'text/plain';
return msg;`

**Comment by helgeerbe** [comment]
OK, mein Fehler. Habe übersehen, dass das gar kein json request ist. Es wird der Parameter "data" als urlencoded Parameter erwartet. Dann ist es so.

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/59169507/5d4b7375-e1f5-4502-adff-00be86e084c4)


````node.js
msg.payload = {};
msg.payload = "data={'serial':'1234567','restart':'true'}";
return msg;
````
![Cursor_und_Node-RED___home_erbehome_de](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/59169507/861c4541-da2f-41ed-9a4b-fe91b06bf5c6)

**Reply by neolauren-cmd** [reply]
MMEEGGGAAAAA, hat geklappt. Danke euch für die Unterstützung.
